experimental/dds/tree2/src/test/shared-tree/fuzz/targetedFuzz.spec.ts

```diff
@@ -103,29 +111,33 @@ describe("Fuzz - Targeted", () => {
 		start: 0,
 		commit: 0,
 	};
+	
 	describe("Composed vs individual changes converge to the same tree", () => {
 		const generatorFactory = () =>
 			takeAsync(opsPerRun, makeOpGenerator(composeVsIndividualWeights));
+		const generator = generatorFactory() as AsyncGenerator<TreeOperation, BranchedTreeFuzzTestState>;
```

Comment:
This change doesn't look necessary, and is potentially harmful: you're evaluating `generatorFactory` to produce a single generator, which once exhausted (that will happen in a single fuzz test) will no longer produce interesting values.

Is the change here motivated by some type-level issue? if so, seems like you should be able to declare `generatorFactory`'s type explicitly to resolve it.

Reply:
Updated to declare `generatoFactory` type instead